presto-main/src/test/java/io/prestosql/operator/BenchmarkScanFilterAndProjectOperator.java

```diff
@@ -216,32 +215,25 @@ private RowExpression getFilter(Type type)
             ImmutableList.Builder<RowExpression> builder = ImmutableList.builder();
             if (type == BIGINT) {
                 for (int i = 0; i < columnCount; i++) {
-                    builder.add(rowExpression("bigint" + i + " + 5", type));
+                    builder.add(rowExpression("bigint" + i + " + 5"));
                 }
             }
             else if (type == VARCHAR) {
                 for (int i = 0; i < columnCount; i++) {
                     // alternatively use identity expression rowExpression("varchar" + i, type) or
                     // rowExpression("substr(varchar" + i + ", 1, 1)", type)
-                    builder.add(rowExpression("concat(varchar" + i + ", 'foo')", type));
+                    builder.add(rowExpression("concat(varchar" + i + ", 'foo')"));
                 }
             }
             return builder.build();
         }
 
-        private RowExpression rowExpression(String expression, Type type)
+        private RowExpression rowExpression(String value)
         {
-            SymbolToInputRewriter symbolToInputRewriter = new SymbolToInputRewriter(sourceLayout);
```

Comment:
Can we remove `SymbolToInputRewriter` completely

Reply:
It’s coming. There are a couple of other dependencies, but I have a change in the complex pushdown PR to do it.